package.json

```diff
@@ -55,6 +55,6 @@
     "karma-coverage": "^2.0.3",
     "karma-jasmine": "^4.0.0",
     "karma-jasmine-html-reporter": "^1.7.0",
-    "typescript": "^4.6.4"
+    "typescript": "4.8"
```

Comment:
I don't think npm likes versions like this.. 
It should be "^4.8.whateverversionisinstalled"